mobile/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { Button, Text, View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type Properties = {
+  isVisible: boolean;
+  onClose: () => void;
+  onDelete: () => void;
+  type?: 'Note';
+};
+
+const Modal: React.FC<Properties> = ({
+  isVisible,
+  onClose,
+  onDelete,
+  type,
+}) => {
+  return (
+    <RNModal visible={isVisible} transparent={true} animationType="none">
+      <View style={styles.modalContainer}>
+        <View style={styles.modalContent}>
+          <Text style={styles.modalTitle}>{`Delete ${type}?`}</Text>
+          <View style={styles.buttonsContainer}>
+            <Button label="Cancel" onPress={onClose} type="modal-cancel" />
+            <Button label="Delete" onPress={onDelete} type="modal-delete" />
+          </View>
+        </View>
+      </View>
+    </RNModal>
+  );
+};
```

Comment:
You created the Modal component specifically for note deletion. Let's create a Modal component that everyone can use later. Right now, I'm working on the 'Add meditation entry functionality #283' task, and I've created this component, so please write me in slack so that I can send you these files, or if you prefer, you can create it yourself)